ci/build_cpp.sh

```diff
@@ -9,9 +9,11 @@ export CMAKE_GENERATOR=Ninja
 
 rapids-print-env
 
+version=$(rapids-generate-version)
```

Comment:
Should we consider just exporting this? Or do you prefer making it an explicit input to the conda build?
```suggestion
export RAPIDS_PACKAGE_VERSION=$(rapids-generate-version)
```

Reply:
I prefer exporting as little as possible because it makes arguments implicit rather than explicit. I prioritize making these scripts as easily readable as possible for non-devops folks.